app/models/concerns/hud_concerns/enrollment.rb

```diff
@@ -22,6 +22,10 @@ module HudConcerns::Enrollment
         where(d_2_end.gteq(d_1_start).or(d_2_end.eq(nil)).and(d_2_start.lteq(d_1_end)))
     end
 
+    scope :ongoing, ->(on_date: Date.current) do
+      open_during_range(on_date..on_date)
```

Comment:
Clients that have an exit date equal to `Date.current` will be included in this scope. Just want to flag that because it has tripped me up so many times. Probalby worth a comment